scripts/packaging/examplesfolderpreupm.ps1

```diff
@@ -0,0 +1,87 @@
+<#
+.SYNOPSIS
+    Prepares the MRTK\Examples folder for UPM packaging.
+.DESCRIPTION
+    Prepares the MRTK\Examples folder for UPM packaging.
+#>
+
+$examplesRoot = "$(Get-Location)"
+
+# This hashtable contains mappings of the sample categories to the folder which contains
+# the code and assets.
+#
+# Note that capitalization below in the key itself is significant.
+#
+# These paths are examplesRoot relative.
+$exampleFolders = [ordered]@{
+    "Demos" = "Demos";
+    "Experimental" = "Experimental";
+    "Editor" = "Editor";

```

Comment:
How is the Editor folder handled? Does it get imported when the corresponding sample (looks like it has an inspector for the dwell sample) is imported?

Reply:
Yep. It is a sample category for things that run in the editor (ex: inspectors)

Reply:
offline confirmation that i was confused by this question. updated the examples layout to fix